.github/workflows/prepare-package-release.yml

```diff
@@ -3,13 +3,57 @@ on:
     workflow_dispatch:
         inputs:
             packages:
-                description: 'Enter a specific package to release, or releases separated by commas, ie @woocommerce/components,@woocommerce/number. Leaving this input blank will prepare to release all eligible packages.'
+                description: 'Enter a specific package to release, or packages separated by commas, ie @woocommerce/components,@woocommerce/number. Leaving this input to the default "-a" will prepare to release all eligible packages.'
                 required: false
                 default: '-a'
 jobs:
     prepare:
-        name: Run the prepare script
+        name: Run prepare script
         runs-on: ubuntu-20.04
         steps:
-            - name: Run
-              run: echo "hello world"
+            - uses: actions/checkout@v3
+
+            - name: Install PNPM
```

Comment:
I gave this a shot in https://github.com/woocommerce/woocommerce/pull/33614/commits/759aabe12c74bdd838d387c448da295375a21c2b

I'm not sure I did it correctly because I ran it twice and each time the build was done without reaching for the cache, taking 7-8 minutes instead of 2-3. This was mostly on the build step, which this workflow doesn't require anyways.